spec/rails_helper.rb

```diff
@@ -73,6 +73,7 @@
   end
 
   VCR.configure do |config|
+    config.allow_http_connections_when_no_cassette = true
```

Comment:
I kept getting an error related to this especially when tests were running on TravisCI, but I stumbled across this as a fix in the VCR docs. Obviously this defaults to false to cut down on API calls, but changing it for this project won't be a problem because the limits for the APIs that are consumed are quite high.